lib/Connectable.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+interface Connectable
+{
```

Comment:
:no_entry_sign: **Error**: Opening brace should be on the same line as the declaration for interface Connectable (*Generic.Classes.OpeningBraceSameLine.BraceOnNewLine*).